views/js/runner/provider/qti.js

```diff
@@ -569,14 +586,22 @@ define([
                         probeOverseer.stop();
                     });
             }
-
-            // prevent the item to be displayed while test runner is destroying
-            if (this.itemRunner) {
-                this.itemRunner.clear();
-            }
-            this.itemRunner = null;
         }
     };
 
+    /**
+     * Displays an exit message, then leaves the runner once the user has acknowledged
+     * @param {runner} runner
+     * @param {Object} data
+     */
+    function leaveRunner(runner, data) {
```

Comment:
I am afraid it is not possible. This function need to be accessible from everywhere in the component, not only from the init() method. However I can move it on top instead of bottom.